napalm/base.py

```diff
@@ -13,6 +13,12 @@
 # the License.
 import napalm.exceptions
 import sys
+import os
+
+from napalm.exceptions import DriverTemplateNotImplemented, TemplateNotImplemented, TemplateRenderException
```

Comment:
You already have in line 13 `import napalm.exceptions` so this one is redundant.